### PR TITLE
BC-1443 - Explicitly define routes in Nuxt

### PR DESCRIPTION
**Pull request by uidp**
# Short Description
BC-1443 - Explicitly define routes in Nuxt

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+				path: "/news/:id([a-z0-9]{24})",
+				component: NewsDetailsPage,
+				name: "news-id",
+			},
+			{
+				path: "/news/new",
+				component: NewsNewPage,
+			},
+			{
+				path: "/news/:id([a-z0-9]{24})/edit",
```
Comment (CeEv) [inline src/router.js:130]: Make it sense to put the id string on const one time on top?

**Review by CeEv — APPROVED** [review]

File: src/serverMiddleware/routes.js [hunk 2]
```diff
@@ -9,11 +9,11 @@ const activationCode = "[a-z0-9]+";
...
-	`^/account/email/edit/?$`,
-	`^/account/name/edit/?$`,
-	`^/account/password/edit/?$`,
-	`^/account/password/edit/?$`,
-	`^/account/locale/edit/?$`,
+	// `^/account/email/edit/?$`,
+	// `^/account/name/edit/?$`,
+	// `^/account/password/edit/?$`,
+	// `^/account/password/edit/?$`,
+	// `^/account/locale/edit/?$`,
```
Comment (odalys-dataport) [inline src/serverMiddleware/routes.js:16]: Shall we go ahead and delete these or are we still unsure of them?

**Review by odalys-dataport — APPROVED** [review]

**Review by CeEv — APPROVED** [review]

File: src/router.js (hunk 1 above)
Comment (uidp) [reply src/router.js]: I changed that.

File: src/serverMiddleware/routes.js (hunk 2 above)
Comment (uidp) [reply src/serverMiddleware/routes.js:16]: I guess we're still not sure. Maybe we have to discuss that with ther Erwin team as well.

File: src/router/routes.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+		component: () => import("../pages/rooms-overview.vue"),
+		name: "rooms-overview",
+	},
+	{
+		path: "/tasks",
+		component: () => import("../pages/tasks/index.vue"),
+		name: "tasks",
+	},
+	{
+		path: "/TermsOfUse",
```
[the remark this answers is not in the document]
Comment (uidp) [reply src/router/routes.js]: fixed that.

**Review by dyedwiper — APPROVED** [review]

**Review by dyedwiper — APPROVED** [review]

**Pull request merged**